sql/logictest: TestLogic failed

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2696549&tab=buildLog) on [master@557346d5fc755db5f64d3e198e47be1f53a4d390](https://github.com/cockroachdb/cockroach/commits/557346d5fc755db5f64d3e198e47be1f53a4d390):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic977178260
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3343: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic977178260
--- FAIL: TestLogic (958.82s)
=== RUN   TestLogic/5node-metadata/distsql_numtables
=== PAUSE TestLogic/5node-metadata/distsql_numtables
=== CONT  TestLogic/5node-metadata/distsql_numtables
=== CONT  TestLogic/5node-metadata/distsql_numtables
    logic.go:2559: 
        
        testdata/logic_test/distsql_numtables:21: 
        expected success, but found
        (XXUUU) cannot up-replicate to s2; missing gossiped StoreDescriptor
--- done: testdata/logic_test/distsql_numtables with config 5node-metadata: 4 tests, 1 failures
E210220 00:01:04.366080 12116468 sql/logictest/logic.go:3575  [-] 1008  pq: cannot up-replicate to s2; missing gossiped StoreDescriptor
    logic.go:3118: 
        testdata/logic_test/distsql_numtables:21: error while processing
    logic.go:3118: pq: cannot up-replicate to s2; missing gossiped StoreDescriptor
--- done: testdata/logic_test/merge_join_dist with config 5node-metadata: 12 tests, 0 failures
        --- FAIL: TestLogic/5node-metadata/distsql_numtables (1.66s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2705518&tab=buildLog) on [master@ab07ae1da6fe154eebc6c6847cca58b30184e14f](https://github.com/cockroachdb/cockroach/commits/ab07ae1da6fe154eebc6c6847cca58b30184e14f):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic233649622
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3343: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic233649622
--- FAIL: TestLogic (740.75s)
=== RUN   TestLogic/5node-metadata/distsql_subquery
=== PAUSE TestLogic/5node-metadata/distsql_subquery
=== CONT  TestLogic/5node-metadata/distsql_subquery
=== CONT  TestLogic/5node-metadata/distsql_subquery
    logic.go:2559: 
        
        testdata/logic_test/distsql_subquery:18: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:43885: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_subquery with config 5node-metadata: 4 tests, 1 failures
E210222 19:54:48.567346 10789584 sql/logictest/logic.go:3575  [-] 1058  pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:43885: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3118: 
        testdata/logic_test/distsql_subquery:18: error while processing
    logic.go:3118: pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:43885: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/5node-metadata/distsql_subquery (1.70s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2705627&tab=buildLog) on [master@bc98209ed36ec47675fe3d7a7d8201b4b77f89ba](https://github.com/cockroachdb/cockroach/commits/bc98209ed36ec47675fe3d7a7d8201b4b77f89ba):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic516258408
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3343: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic516258408
--- FAIL: TestLogic (1031.64s)
=== RUN   TestLogic/fakedist-spec-planning/errors
=== PAUSE TestLogic/fakedist-spec-planning/errors
=== CONT  TestLogic/fakedist-spec-planning/errors
E210222 20:27:49.956148 10856322 jobs/adopt.go:117  [-] 7525  could not run claimed job 635552627370852353: job 635552627370852353: status changed to succeeded which is not resumable`
E210222 20:27:49.986725 10857012 jobs/adopt.go:117  [-] 7526  could not run claimed job 635552627323830273: job 635552627323830273: status changed to succeeded which is not resumable`
E210222 20:27:50.008918 10857561 jobs/adopt.go:117  [-] 7527  could not run claimed job 635552627463684097: job 635552627463684097: status changed to succeeded which is not resumable`
--- progress: testdata/logic_test/show_create_all_tables: 9 statements/queries
E210222 20:27:50.032915 10858483 jobs/adopt.go:117  [-] 7528  could not run claimed job 635552627347259393: job 635552627347259393: status changed to succeeded which is not resumable`
E210222 20:27:50.034966 10858482 jobs/adopt.go:117  [-] 7529  could not run claimed job 635552627423838209: job 635552627423838209: status changed to succeeded which is not resumable`
=== CONT  TestLogic/fakedist-spec-planning/errors
    logic.go:3122: 
        testdata/logic_test/errors:19: error while processing
    logic.go:3122: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n3 at 127.0.0.1:34735: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/system with config fakedist-spec-planning: 65 tests, 0 failures
        --- FAIL: TestLogic/fakedist-spec-planning/errors (1.85s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.72s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716115&tab=buildLog) on [master@5cd07e02c291d017bedd2a237956a62eb63dbcc9](https://github.com/cockroachdb/cockroach/commits/5cd07e02c291d017bedd2a237956a62eb63dbcc9):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic237413412
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic237413412
--- FAIL: TestLogic (1294.75s)
=== RUN   TestLogic/fakedist-spec-planning/show_indexes
=== PAUSE TestLogic/fakedist-spec-planning/show_indexes
=== CONT  TestLogic/fakedist-spec-planning/show_indexes
E210225 01:17:46.717279 11401034 jobs/adopt.go:117  [-] 7559  could not run claimed job 636175865142411265: job 636175865142411265: status changed to succeeded which is not resumable`
E210225 01:17:46.734043 11401736 jobs/adopt.go:117  [-] 7560  could not run claimed job 636175865086050305: job 636175865086050305: status changed to succeeded which is not resumable`
E210225 01:17:46.739681 11402071 jobs/adopt.go:117  [-] 7561  could not run claimed job 636175865136676865: job 636175865136676865: status changed to succeeded which is not resumable`
E210225 01:17:46.756434 11402182 kv/kvclient/kvcoord/txn_interceptor_committer.go:449  [-] 7562  making txn commit explicit failed for "unnamed" meta={id=39096c6e pri=0.05947514 epo=0 ts=1614215866.731093780,0 min=1614215866.731093780,0 seq=2} lock=true stat=STAGING rts=1614215866.731093780,0 wto=false gul=1614215867.231093780,0 int=1 ifw=1: result is ambiguous (server shutdown)
=== CONT  TestLogic/fakedist-spec-planning/show_indexes
    testcluster.go:1439: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/fakedist-spec-planning/show_indexes (1.54s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.66s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721032&tab=buildLog) on [master@ac25d9f3511435c4e6962f71eb169fd6d7d0ef28](https://github.com/cockroachdb/cockroach/commits/ac25d9f3511435c4e6962f71eb169fd6d7d0ef28):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic596100037
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic596100037
--- FAIL: TestLogic (752.70s)
=== RUN   TestLogic/5node/distsql_distinct_on
=== PAUSE TestLogic/5node/distsql_distinct_on
=== CONT  TestLogic/5node/distsql_distinct_on
=== CONT  TestLogic/5node/distsql_distinct_on
    logic.go:2567: 
        
        testdata/logic_test/distsql_distinct_on:40: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:44201: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_distinct_on with config 5node: 5 tests, 1 failures
E210226 02:36:17.364026 10710610 sql/logictest/logic.go:3583  [-] 7053  pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:44201: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3126: 
        testdata/logic_test/distsql_distinct_on:40: error while processing
    logic.go:3126: pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:44201: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/5node/distsql_distinct_on (2.08s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726301&tab=buildLog) on [master@dcc8ab04251aa954f086cd2b347cd1d1b976a312](https://github.com/cockroachdb/cockroach/commits/dcc8ab04251aa954f086cd2b347cd1d1b976a312):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic228119198
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic228119198
--- FAIL: TestLogic (971.83s)
=== RUN   TestLogic/fakedist-spec-planning/txn_stats
=== PAUSE TestLogic/fakedist-spec-planning/txn_stats
=== CONT  TestLogic/fakedist-spec-planning/txn_stats
=== CONT  TestLogic/fakedist-spec-planning/txn_stats
    logic.go:3130: 
        testdata/logic_test/txn_stats:44: error while processing
    logic.go:3130: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n2 at 127.0.0.1:33081: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/fakedist-spec-planning/txn_stats (1.39s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.58s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2730013&tab=buildLog) on [master@6521a8e138c9838853f9502303faf99da69aad2d](https://github.com/cockroachdb/cockroach/commits/6521a8e138c9838853f9502303faf99da69aad2d):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic409701549
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic409701549
--- FAIL: TestLogic (998.67s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.56s)
=== RUN   TestLogic/fakedist-spec-planning/alter_sequence
=== PAUSE TestLogic/fakedist-spec-planning/alter_sequence
=== CONT  TestLogic/fakedist-spec-planning/alter_sequence
--- done: testdata/logic_test/save_table with config fakedist-spec-planning: 21 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/alter_sequence
    testcluster.go:1439: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/fakedist-spec-planning/alter_sequence (1.43s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733481&tab=buildLog) on [master@ae1e417e061cc1b869927959e8830e0e5cde2622](https://github.com/cockroachdb/cockroach/commits/ae1e417e061cc1b869927959e8830e0e5cde2622):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic044184690
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic044184690
--- FAIL: TestLogic (988.19s)
=== RUN   TestLogic/fakedist-spec-planning/ordinality
=== PAUSE TestLogic/fakedist-spec-planning/ordinality
=== CONT  TestLogic/fakedist-spec-planning/ordinality
--- done: testdata/logic_test/partial_index with config fakedist-spec-planning: 316 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/ordinality
    logic.go:3130: 
        testdata/logic_test/ordinality:102: error while processing
    logic.go:3130: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n2 at 127.0.0.1:43669: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210302 05:25:10.426834 11625673 sql/stats/automatic_stats.go:302  [-] 1469  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/fakedist-spec-planning/ordinality (1.50s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.55s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2749097&tab=buildLog) on [master@f0c2d8a2f762ee33ca044ae86f46d06fc35682db](https://github.com/cockroachdb/cockroach/commits/f0c2d8a2f762ee33ca044ae86f46d06fc35682db):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic976886314
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic976886314
--- FAIL: TestLogic (949.79s)
=== RUN   TestLogic/fakedist-vec-off/inverted_join_geospatial
=== PAUSE TestLogic/fakedist-vec-off/inverted_join_geospatial
=== CONT  TestLogic/fakedist-vec-off/inverted_join_geospatial
--- done: testdata/logic_test/jobs with config fakedist-vec-off: 21 tests, 0 failures
--- done: testdata/logic_test/txn_retry with config fakedist-vec-off: 6 tests, 0 failures
--- done: testdata/logic_test/txn_as_of with config fakedist-vec-off: 72 tests, 0 failures
E210305 14:21:54.181701 5921621 jobs/adopt.go:260  [n1] 856  job 638594954554540033: adoption completed with error job 638594954554540033: could not mark as reverting: log-job: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer: log-job: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- total progress: 75201 statements/queries
=== CONT  TestLogic/fakedist-vec-off/inverted_join_geospatial
    logic.go:2567: 
        
        testdata/logic_test/inverted_join_geospatial:33: SELECT lk, rk FROM ltable JOIN rtable@geom_index ON ST_Intersects(ltable.geom1, rtable.geom) ORDER BY (lk, rk)
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:40659: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210305 14:21:55.397713 5324437 sql/logictest/logic.go:3558  [-] 858  
E210305 14:21:55.397713 5324437 sql/logictest/logic.go:3558  [-] 858 +pq: failed to connect to n2 at 127.0.0.1:40659: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2300: 
         pq: failed to connect to n2 at 127.0.0.1:40659: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/inverted_join_geospatial with config fakedist-vec-off: 5 tests, 2 failures
E210305 14:21:55.398118 5324437 sql/logictest/logic.go:3583  [-] 859  testdata/logic_test/inverted_join_geospatial:45: too many errors encountered, skipping the rest of the input
    logic.go:3126: 
        testdata/logic_test/inverted_join_geospatial:45: error while processing
    logic.go:3126: testdata/logic_test/inverted_join_geospatial:45: too many errors encountered, skipping the rest of the input
--- progress: testdata/logic_test/join: 113 statements/queries
        --- FAIL: TestLogic/fakedist-vec-off/inverted_join_geospatial (1.41s)
=== RUN   TestLogic/fakedist-vec-off
    --- FAIL: TestLogic/fakedist-vec-off (0.54s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2753663&tab=buildLog) on [master@a69e6549a71f5a0e83eb13509001f4d7351050fb](https://github.com/cockroachdb/cockroach/commits/a69e6549a71f5a0e83eb13509001f4d7351050fb):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic150275333
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic150275333
--- FAIL: TestLogic (1873.11s)
=== RUN   TestLogic/fakedist-metadata/alter_primary_key/create_table_change_pk
=== CONT  TestLogic/fakedist-metadata/alter_primary_key/create_table_change_pk
    logic.go:2567: 
        
        testdata/logic_test/alter_primary_key:1332: 
        expected success, but found
        (42703) column "crdb_internal_id_shard_10" does not exist
        column_resolver.go:206: in NewUndefinedColumnError()
E210307 08:35:42.725685 7777196 sql/logictest/logic.go:3558  [-] 821  
E210307 08:35:42.725685 7777196 sql/logictest/logic.go:3558  [-] 821 +pq: column "crdb_internal_id_shard_10" does not exist
    logic.go:1844: 
         pq: column "crdb_internal_id_shard_10" does not exist
--- done: testdata/logic_test/alter_primary_key with config fakedist-metadata: 227 tests, 2 failures
--- progress: testdata/logic_test/alter_table: 300 statements/queries
--- total progress: 102482 statements/queries
            --- FAIL: TestLogic/fakedist-metadata/alter_primary_key/create_table_change_pk (2.55s)
=== RUN   TestLogic/fakedist-metadata/alter_primary_key
=== PAUSE TestLogic/fakedist-metadata/alter_primary_key
=== CONT  TestLogic/fakedist-metadata/alter_primary_key
--- progress: testdata/logic_test/alter_table: 68 statements/queries
E210307 08:35:34.903290 7753854 jobs/adopt.go:260  [n1] 806  job 639093095472267265: adoption completed with error validation of CHECK "i < 0:::INT8" failed on row: a=1, x=NULL, y=1.3, z=1.4, d=NULL, f=NULL, g=1, h=NULL, i=1
E210307 08:35:35.153586 7754777 jobs/adopt.go:260  [n1] 807  job 639093096392097793: adoption completed with error validation of CHECK "i < g" failed on row: a=1, x=NULL, y=1.3, z=1.4, d=NULL, f=NULL, g=1, h=NULL, i=1
        --- FAIL: TestLogic/fakedist-metadata/alter_primary_key (9.63s)
=== RUN   TestLogic/fakedist-metadata
    --- FAIL: TestLogic/fakedist-metadata (0.59s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=2

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2757312&tab=buildLog) on [master@89c379950cb4ca77c5d25e45aac4008b1f1758d9](https://github.com/cockroachdb/cockroach/commits/89c379950cb4ca77c5d25e45aac4008b1f1758d9):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic914265095
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic914265095
--- FAIL: TestLogic (752.85s)
=== RUN   TestLogic/fakedist-spec-planning/alter_primary_key/create_table_change_pk
=== CONT  TestLogic/fakedist-spec-planning/alter_primary_key/create_table_change_pk
    logic.go:2567: 
        
        testdata/logic_test/alter_primary_key:1332: 
        expected success, but found
        (42703) column "crdb_internal_id_shard_10" does not exist
        column_resolver.go:206: in NewUndefinedColumnError()
E210309 05:57:07.925464 10761106 sql/logictest/logic.go:3558  [-] 1191  
E210309 05:57:07.925464 10761106 sql/logictest/logic.go:3558  [-] 1191 +pq: column "crdb_internal_id_shard_10" does not exist
    logic.go:1844: 
         pq: column "crdb_internal_id_shard_10" does not exist
--- done: testdata/logic_test/alter_primary_key with config fakedist-spec-planning: 227 tests, 2 failures
--- done: testdata/logic_test/interleaved with config fakedist-spec-planning: 99 tests, 0 failures
            --- FAIL: TestLogic/fakedist-spec-planning/alter_primary_key/create_table_change_pk (4.71s)
=== RUN   TestLogic/fakedist-spec-planning/alter_primary_key
=== PAUSE TestLogic/fakedist-spec-planning/alter_primary_key
=== CONT  TestLogic/fakedist-spec-planning/alter_primary_key
--- progress: testdata/logic_test/alter_type: 86 statements/queries
        --- FAIL: TestLogic/fakedist-spec-planning/alter_primary_key (18.49s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.48s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2760329&tab=buildLog) on [master@dd87d114508b8b34230ae7da560ab053dd4f58ce](https://github.com/cockroachdb/cockroach/commits/dd87d114508b8b34230ae7da560ab053dd4f58ce):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic927282679
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic927282679
--- FAIL: TestLogic (754.47s)
=== RUN   TestLogic/5node/distsql_datetime
=== PAUSE TestLogic/5node/distsql_datetime
=== CONT  TestLogic/5node/distsql_datetime
=== CONT  TestLogic/5node/distsql_datetime
    logic.go:2557: 
        
        testdata/logic_test/distsql_datetime:3: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:44901: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_datetime with config 5node: 0 tests, 1 failures
E210309 21:22:34.947860 11118492 sql/logictest/logic.go:3573  [-] 1209  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:44901: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3116: 
        testdata/logic_test/distsql_datetime:3: error while processing
    logic.go:3116: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:44901: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/5node/distsql_datetime (2.54s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2762459&tab=buildLog) on [master@3bae381e45397225ff926f3f8f60d52b70fdc661](https://github.com/cockroachdb/cockroach/commits/3bae381e45397225ff926f3f8f60d52b70fdc661):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic961305549
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic961305549
--- FAIL: TestLogic (1554.07s)
=== RUN   TestLogic/local-vec-off/alter_primary_key/create_table_change_pk
E210310 09:07:38.128535 1719914 jobs/adopt.go:260  [n1] 190  job 639948744322187265: adoption completed with error could not parse "hello" as type int: strconv.ParseInt: parsing "hello": invalid syntax
E210310 09:07:38.542246 1721757 jobs/adopt.go:260  [n1] 191  job 639948745763160065: adoption completed with error could not parse "a\\01" as type bytes: bytea encoded value ends with incomplete escape sequence
--- progress: testdata/logic_test/alter_column_type: 115 statements/queries
E210310 09:07:38.790098 1722614 jobs/adopt.go:260  [n1] 192  job 639948746472423425: adoption completed with error integer out of range for type int2
--- done: testdata/logic_test/alter_column_type with config local-vec-off: 122 tests, 0 failures
--- total progress: 29593 statements/queries
=== CONT  TestLogic/local-vec-off/alter_primary_key/create_table_change_pk
    logic.go:2557: 
        
        testdata/logic_test/alter_primary_key:1332: 
        expected success, but found
        (42703) column "crdb_internal_id_shard_10" does not exist
        column_resolver.go:206: in NewUndefinedColumnError()
E210310 09:07:40.213479 1719775 sql/logictest/logic.go:3548  [-] 196  
E210310 09:07:40.213479 1719775 sql/logictest/logic.go:3548  [-] 196 +pq: column "crdb_internal_id_shard_10" does not exist
    logic.go:1834: 
         pq: column "crdb_internal_id_shard_10" does not exist
--- done: testdata/logic_test/alter_primary_key with config local-vec-off: 227 tests, 2 failures
            --- FAIL: TestLogic/local-vec-off/alter_primary_key/create_table_change_pk (2.22s)
=== RUN   TestLogic/local-vec-off/alter_primary_key
=== PAUSE TestLogic/local-vec-off/alter_primary_key
=== CONT  TestLogic/local-vec-off/alter_primary_key
        --- FAIL: TestLogic/local-vec-off/alter_primary_key (8.17s)
=== RUN   TestLogic/local-vec-off
    --- FAIL: TestLogic/local-vec-off (0.33s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=2

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2764508&tab=buildLog) on [master@cd96b8663d48c02633a4e43dda83e6dbf41362d3](https://github.com/cockroachdb/cockroach/commits/cd96b8663d48c02633a4e43dda83e6dbf41362d3):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic765069083
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic765069083
--- FAIL: TestLogic (971.35s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)
=== RUN   TestLogic/5node-metadata/distsql_datetime
=== PAUSE TestLogic/5node-metadata/distsql_datetime
=== CONT  TestLogic/5node-metadata/distsql_datetime
=== CONT  TestLogic/5node-metadata/distsql_datetime
    logic.go:2557: 
        
        testdata/logic_test/distsql_datetime:3: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:39631: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_datetime with config 5node-metadata: 0 tests, 1 failures
E210310 21:48:59.038509 12911254 sql/logictest/logic.go:3573  [-] 1267  pq: while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:39631: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3116: 
        testdata/logic_test/distsql_datetime:3: error while processing
    logic.go:3116: pq: while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:39631: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- total progress: 137174 statements/queries
        --- FAIL: TestLogic/5node-metadata/distsql_datetime (2.03s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2771534&tab=buildLog) on [master@e7f33b7ef323bd29ba203c362408776bd77eac52](https://github.com/cockroachdb/cockroach/commits/e7f33b7ef323bd29ba203c362408776bd77eac52):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic451763382
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic451763382
--- FAIL: TestLogic (1085.75s)
=== RUN   TestLogic/5node-disk
    --- FAIL: TestLogic/5node-disk (0.00s)
=== RUN   TestLogic/5node-disk/distsql_merge_join
=== PAUSE TestLogic/5node-disk/distsql_merge_join
=== CONT  TestLogic/5node-disk/distsql_merge_join
=== CONT  TestLogic/5node-disk/distsql_merge_join
    testcluster.go:1442: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/distsql_subquery with config 5node-disk: 7 tests, 0 failures
--- done: testdata/logic_test/contention_event with config 5node-disk: 11 tests, 0 failures
--- total progress: 137658 statements/queries
        --- FAIL: TestLogic/5node-disk/distsql_merge_join (2.06s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2771751&tab=buildLog) on [master@e7f33b7ef323bd29ba203c362408776bd77eac52](https://github.com/cockroachdb/cockroach/commits/e7f33b7ef323bd29ba203c362408776bd77eac52):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic161582333
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic161582333
--- FAIL: TestLogic (800.95s)
=== RUN   TestLogic/5node-spec-planning/distsql_union
=== PAUSE TestLogic/5node-spec-planning/distsql_union
=== CONT  TestLogic/5node-spec-planning/distsql_union
=== CONT  TestLogic/5node-spec-planning/distsql_union
    logic.go:2557: 
        
        testdata/logic_test/distsql_union:22: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:35645: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_union with config 5node-spec-planning: 3 tests, 1 failures
E210312 17:31:56.937224 11643012 sql/logictest/logic.go:3573  [-] 1173  pq: while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:35645: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3116: 
        testdata/logic_test/distsql_union:22: error while processing
    logic.go:3116: pq: while carrying out changes [{ADD_VOTER n5,s5}]: failed to connect to n5 at 127.0.0.1:35645: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- total progress: 138082 statements/queries
--- done: testdata/logic_test/inverted_join_geospatial_dist with config 5node-spec-planning: 29 tests, 0 failures
--- done: testdata/logic_test/merge_join_dist with config 5node-spec-planning: 12 tests, 0 failures
--- done: testdata/logic_test/distsql_datetime with config 5node-spec-planning: 2 tests, 0 failures
--- done: testdata/logic_test/inverted_join_multi_column_dist with config 5node-spec-planning: 16 tests, 0 failures
--- done: testdata/logic_test/distsql_numtables with config 5node-spec-planning: 27 tests, 0 failures
E210312 17:31:57.863771 11676958 sql/stats/automatic_stats.go:302  [-] 1174  failed to refresh stats: node unavailable; try another peer
--- done: testdata/logic_test/distsql_tighten_spans with config 5node-spec-planning: 40 tests, 0 failures
E210312 17:31:58.042815 11740492 jobs/adopt.go:260  [n1] 1175  job 640614131390775297: adoption completed with error job 640614131390775297: could not mark as reverting: load-job-query: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer: log-job: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- done: testdata/logic_test/inverted_join_json_array_dist with config 5node-spec-planning: 26 tests, 0 failures
--- done: testdata/logic_test/distsql_distinct_on with config 5node-spec-planning: 15 tests, 0 failures
E210312 17:31:58.431058 11745362 jobs/adopt.go:260  [n1] 1176  job 640614132657651713: adoption completed with error job-update: collecting referenced descriptors: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- progress: testdata/logic_test/distsql_agg: 19 statements/queries
--- progress: testdata/logic_test/hash_join_dist: 14 statements/queries
--- progress: testdata/logic_test/distsql_merge_join: 27 statements/queries
--- done: testdata/logic_test/hash_join_dist with config 5node-spec-planning: 18 tests, 0 failures
E210312 17:31:59.608843 11756550 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 1177  making txn commit explicit failed for "unnamed" meta={id=340f3101 pri=0.03129016 epo=0 ts=1615570319.602523681,0 min=1615570319.602523681,0 seq=2} lock=true stat=STAGING rts=1615570319.602523681,0 wto=false gul=1615570320.102523681,0 int=1 ifw=1: result is ambiguous (server shutdown)
--- total progress: 138278 statements/queries
        --- FAIL: TestLogic/5node-spec-planning/distsql_union (2.20s)
=== RUN   TestLogic/5node-spec-planning
    --- FAIL: TestLogic/5node-spec-planning (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2773434&tab=buildLog) on [master@283a21e3e84037b03ea73ca7d8cba6a9b6c11f62](https://github.com/cockroachdb/cockroach/commits/283a21e3e84037b03ea73ca7d8cba6a9b6c11f62):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic047338459
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic047338459
--- FAIL: TestLogic (972.29s)
=== RUN   TestLogic/5node-spec-planning/inverted_join_multi_column_dist
=== PAUSE TestLogic/5node-spec-planning/inverted_join_multi_column_dist
=== CONT  TestLogic/5node-spec-planning/inverted_join_multi_column_dist
=== CONT  TestLogic/5node-spec-planning/inverted_join_multi_column_dist
    logic.go:2557: 
        
        testdata/logic_test/inverted_join_multi_column_dist:76: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:45933: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/inverted_join_multi_column_dist with config 5node-spec-planning: 5 tests, 1 failures
E210313 02:41:10.689646 13253608 sql/logictest/logic.go:3573  [-] 1274  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:45933: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3116: 
        testdata/logic_test/inverted_join_multi_column_dist:76: error while processing
    logic.go:3116: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:45933: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/distsql_subquery with config 5node-spec-planning: 7 tests, 0 failures
--- done: testdata/logic_test/contention_event with config 5node-spec-planning: 12 tests, 0 failures
--- done: testdata/logic_test/inverted_join_geospatial_dist with config 5node-spec-planning: 29 tests, 0 failures
        --- FAIL: TestLogic/5node-spec-planning/inverted_join_multi_column_dist (2.22s)
=== RUN   TestLogic/5node-spec-planning
    --- FAIL: TestLogic/5node-spec-planning (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2773472&tab=buildLog) on [master@283a21e3e84037b03ea73ca7d8cba6a9b6c11f62](https://github.com/cockroachdb/cockroach/commits/283a21e3e84037b03ea73ca7d8cba6a9b6c11f62):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic296668288
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3341: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic296668288
--- FAIL: TestLogic (957.38s)
=== RUN   TestLogic/fakedist-spec-planning/inflight_trace_spans
=== PAUSE TestLogic/fakedist-spec-planning/inflight_trace_spans
=== CONT  TestLogic/fakedist-spec-planning/inflight_trace_spans
=== CONT  TestLogic/fakedist-spec-planning/inflight_trace_spans
    logic.go:2331: let $curr_trace_id = 1967622793753616883
=== CONT  TestLogic/fakedist-spec-planning/inflight_trace_spans
    logic.go:2290: 
         
        testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
          FROM current_trace_spans
        expected:
            true
        but found (query options: "") :
            false
--- done: testdata/logic_test/inflight_trace_spans with config fakedist-spec-planning: 9 tests, 1 failures
        --- FAIL: TestLogic/fakedist-spec-planning/inflight_trace_spans (0.98s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.68s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2801771&tab=buildLog) on [master@7899744d137334e8337e3bc3122f816c141ed7b5](https://github.com/cockroachdb/cockroach/commits/7899744d137334e8337e3bc3122f816c141ed7b5):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic479913299
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic479913299
--- FAIL: TestLogic (1016.61s)
=== RUN   TestLogic/5node/merge_join_dist_vec
=== PAUSE TestLogic/5node/merge_join_dist_vec
=== CONT  TestLogic/5node/merge_join_dist_vec
=== CONT  TestLogic/5node/merge_join_dist_vec
    logic.go:2549: 
        
        testdata/logic_test/merge_join_dist_vec:23: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:43635: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/merge_join_dist_vec with config 5node: 6 tests, 1 failures
E210322 22:49:09.664515 12843841 sql/logictest/logic.go:3565  [-] 1334  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:43635: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
    logic.go:3108: 
        testdata/logic_test/merge_join_dist_vec:23: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:43635: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        --- FAIL: TestLogic/5node/merge_join_dist_vec (2.08s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2802569&tab=buildLog) on [master@7826ab1dabcfaa2bad8e84025351387e17ec0e5d](https://github.com/cockroachdb/cockroach/commits/7826ab1dabcfaa2bad8e84025351387e17ec0e5d):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic195117160
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic195117160
--- FAIL: TestLogic (1054.53s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.58s)
=== RUN   TestLogic/5node-spec-planning/merge_join_dist
=== PAUSE TestLogic/5node-spec-planning/merge_join_dist
=== CONT  TestLogic/5node-spec-planning/merge_join_dist
=== CONT  TestLogic/5node-spec-planning/merge_join_dist
    logic.go:2549: 
        
        testdata/logic_test/merge_join_dist:23: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36979: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/merge_join_dist with config 5node-spec-planning: 6 tests, 1 failures
E210323 02:19:41.121166 13370297 sql/logictest/logic.go:3565  [-] 1329  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36979: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3108: 
        testdata/logic_test/merge_join_dist:23: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36979: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/contention_event with config 5node-spec-planning: 12 tests, 0 failures
--- total progress: 139486 statements/queries
        --- FAIL: TestLogic/5node-spec-planning/merge_join_dist (1.92s)
=== RUN   TestLogic/5node-spec-planning
    --- FAIL: TestLogic/5node-spec-planning (0.00s)
=== RUN   TestLogic/fakedist-spec-planning/collatedstring_uniqueindex2
=== PAUSE TestLogic/fakedist-spec-planning/collatedstring_uniqueindex2
=== CONT  TestLogic/fakedist-spec-planning/collatedstring_uniqueindex2
=== CONT  TestLogic/fakedist-spec-planning/collatedstring_uniqueindex2
    logic.go:2549: 
        
        testdata/logic_test/collatedstring_uniqueindex2:29: SELECT a, b FROM t ORDER BY a, b
        expected success, but found
        (XXUUU) failed to connect to n3 at 127.0.0.1:38647: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210323 02:18:07.492930 11089452 sql/logictest/logic.go:3540  [-] 1198  
E210323 02:18:07.492930 11089452 sql/logictest/logic.go:3540  [-] 1198 +pq: failed to connect to n3 at 127.0.0.1:38647: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2282: 
         pq: failed to connect to n3 at 127.0.0.1:38647: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/collatedstring_uniqueindex2 with config fakedist-spec-planning: 4 tests, 2 failures
E210323 02:18:07.493435 11089452 sql/logictest/logic.go:3565  [-] 1199  testdata/logic_test/collatedstring_uniqueindex2:42: too many errors encountered, skipping the rest of the input
    logic.go:3108: 
        testdata/logic_test/collatedstring_uniqueindex2:42: error while processing
    logic.go:3108: testdata/logic_test/collatedstring_uniqueindex2:42: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist-spec-planning/collatedstring_uniqueindex2 (1.50s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2806904&tab=buildLog) on [master@ab7362161d5e3b88f4730db94b95c5926b5eb3af](https://github.com/cockroachdb/cockroach/commits/ab7362161d5e3b88f4730db94b95c5926b5eb3af):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic726572008
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic726572008
--- FAIL: TestLogic (802.79s)
=== RUN   TestLogic/5node/hash_join_dist
=== PAUSE TestLogic/5node/hash_join_dist
=== CONT  TestLogic/5node/hash_join_dist
E210323 22:21:40.427406 11109734 sql/stats/automatic_stats.go:302  [-] 1235  failed to refresh stats: node unavailable; try another peer
E210323 22:21:40.427758 11140263 sql/stats/automatic_stats.go:302  [-] 1236  failed to refresh stats: node unavailable; try another peer
=== CONT  TestLogic/5node/hash_join_dist
    logic.go:2549: 
        
        testdata/logic_test/hash_join_dist:9: 
        expected success, but found
        (XXUUU) cannot up-replicate to s3; missing gossiped StoreDescriptor
--- done: testdata/logic_test/hash_join_dist with config 5node: 2 tests, 1 failures
E210323 22:21:42.141639 11100659 sql/logictest/logic.go:3565  [-] 1241  pq: cannot up-replicate to s3; missing gossiped StoreDescriptor
    logic.go:3108: 
        testdata/logic_test/hash_join_dist:9: error while processing
    logic.go:3108: pq: cannot up-replicate to s3; missing gossiped StoreDescriptor
--- progress: testdata/logic_test/ranges: 20 statements/queries
--- done: testdata/logic_test/merge_join_dist with config 5node: 12 tests, 0 failures
--- done: testdata/logic_test/inverted_filter_geospatial_dist with config 5node: 25 tests, 0 failures
        --- FAIL: TestLogic/5node/hash_join_dist (1.77s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2808712&tab=buildLog) on [master@9fa4b125bfb07552b43ba4fd52c9301afd7a937b](https://github.com/cockroachdb/cockroach/commits/9fa4b125bfb07552b43ba4fd52c9301afd7a937b):

```
        expected success, but found
        (XX000) internal error: unexpected error from the vectorized engine: runtime error: index out of range [1] with length 1
        error.go:88: in func1()
        DETAIL: stack trace:
        github.com/cockroachdb/cockroach/pkg/sql/colexecerror/error.go:88: func1()
        runtime/panic.go:969: gopanic()
        runtime/panic.go:88: goPanicIndex()
        github.com/cockroachdb/cockroach/pkg/col/coldata/nulls.go:181: NullAt()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecjoin/crossjoiner.eg.go:740: func1()
        github.com/cockroachdb/cockroach/pkg/sql/colmem/allocator.go:297: PerformOperation()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecjoin/crossjoiner.eg.go:51: buildFromLeftInput()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecjoin/crossjoiner.go:110: Next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecutils/operator.go:145: Next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecbase/const.eg.go:212: Next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/colexecbase/simple_project.go:124: Next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/invariants_checker.go:78: Next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/materializer.go:290: next()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/materializer.go:315: nextAdapter()
        github.com/cockroachdb/cockroach/pkg/sql/colexecerror/error.go:91: CatchVectorizedRuntimeError()
        github.com/cockroachdb/cockroach/pkg/sql/colexec/materializer.go:321: Next()
        github.com/cockroachdb/cockroach/pkg/sql/execinfra/base.go:171: Run()
        github.com/cockroachdb/cockroach/pkg/sql/execinfra/processorsbase.go:774: Run()
        github.com/cockroachdb/cockroach/pkg/sql/flowinfra/flow.go:382: Run()
        github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:359: Run()
        github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:968: PlanAndRun()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1040: execWithDistSQLEngine()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:912: dispatchToExecutionEngine()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:656: execStmtInOpenState()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:120: execStmt()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1515: func1()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1517: execCmd()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1438: run()
        github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:482: ServeConn()
        github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go:626: func1()
        runtime/asm_amd64.s:1374: goexit()
        
        NOTE: internal errors may have more details in logs. Use -show-logs.
E210324 07:58:01.762364 8097184 sql/logictest/logic.go:3540  [-] 931  
E210324 07:58:01.762364 8097184 sql/logictest/logic.go:3540  [-] 931 +pq: internal error: unexpected error from the vectorized engine: runtime error: index out of range [1] with length 1
    logic.go:2282: 
         pq: internal error: unexpected error from the vectorized engine: runtime error: index out of range [1] with length 1
--- done: testdata/logic_test/postgresjoin with config fakedist-disk: 31 tests, 2 failures
E210324 07:58:01.762578 8097184 sql/logictest/logic.go:3565  [-] 932  testdata/logic_test/postgresjoin:454: too many errors encountered, skipping the rest of the input
    logic.go:3108: 
        testdata/logic_test/postgresjoin:454: error while processing
    logic.go:3108: testdata/logic_test/postgresjoin:454: too many errors encountered, skipping the rest of the input
E210324 07:58:01.768962 9170999 sql/stats/automatic_stats.go:302  [-] 933  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/fakedist-disk/postgresjoin (0.51s)
=== RUN   TestLogic/fakedist-disk
    --- FAIL: TestLogic/fakedist-disk (0.59s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=2

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2811106&tab=buildLog) on [master@f84a9da9e62bad67c6a72d6c0717cd87e2372357](https://github.com/cockroachdb/cockroach/commits/f84a9da9e62bad67c6a72d6c0717cd87e2372357):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic698830742
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic698830742
--- FAIL: TestLogic (957.45s)
=== RUN   TestLogic/fakedist-spec-planning/geospatial_zm
=== PAUSE TestLogic/fakedist-spec-planning/geospatial_zm
=== CONT  TestLogic/fakedist-spec-planning/geospatial_zm
=== CONT  TestLogic/fakedist-spec-planning/geospatial_zm
    testcluster.go:1442: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/fakedist-spec-planning/geospatial_zm (1.49s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.51s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2814762&tab=buildLog) on [master@aaac3aeae88c5864643881c7d74e24074ffb2c86](https://github.com/cockroachdb/cockroach/commits/aaac3aeae88c5864643881c7d74e24074ffb2c86):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic459801672
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic459801672
--- FAIL: TestLogic (1002.68s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.66s)
=== RUN   TestLogic/fakedist-spec-planning/pg_extension
=== PAUSE TestLogic/fakedist-spec-planning/pg_extension
=== CONT  TestLogic/fakedist-spec-planning/pg_extension
--- done: testdata/logic_test/postgresjoin with config fakedist-spec-planning: 101 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/pg_extension
    logic.go:3112: 
        testdata/logic_test/pg_extension:55: error while processing
    logic.go:3112: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n3 at 127.0.0.1:40399: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210325 19:15:05.441231 11861640 sql/stats/automatic_stats.go:302  [-] 1202  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/fakedist-spec-planning/pg_extension (1.53s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2816056&tab=buildLog) on [master@1c9289e0589bf46aa5205290036b672d9afc525e](https://github.com/cockroachdb/cockroach/commits/1c9289e0589bf46aa5205290036b672d9afc525e):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic134629110
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic134629110
--- FAIL: TestLogic (1020.11s)
=== RUN   TestLogic/5node-spec-planning
    --- FAIL: TestLogic/5node-spec-planning (0.00s)
=== RUN   TestLogic/5node-spec-planning/inverted_join_json_array_dist
=== PAUSE TestLogic/5node-spec-planning/inverted_join_json_array_dist
=== CONT  TestLogic/5node-spec-planning/inverted_join_json_array_dist
--- done: testdata/logic_test/distsql_subquery with config 5node-spec-planning: 7 tests, 0 failures
E210326 02:44:36.453923 13252575 sql/logictest/logic.go:3553  [-] 1321  
E210326 02:44:36.453923 13252575 sql/logictest/logic.go:3553  [-] 1321 +testdata/logic_test/inverted_join_json_array_dist:60: 
E210326 02:44:36.453923 13252575 sql/logictest/logic.go:3553  [-] 1321 +expected success, but found
E210326 02:44:36.453923 13252575 sql/logictest/logic.go:3553  [-] 1321 +(XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39745: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210326 02:44:36.453923 13252575 sql/logictest/logic.go:3553  [-] 1321 +nodedialer.go:169: in dial()
    logic.go:2549: 
        
        testdata/logic_test/inverted_join_json_array_dist:60: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39745: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/inverted_join_json_array_dist with config 5node-spec-planning: 6 tests, 1 failures
E210326 02:44:36.454376 13252575 sql/logictest/logic.go:3565  [-] 1322  pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39745: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3108: 
        testdata/logic_test/inverted_join_json_array_dist:60: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39745: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/inverted_join_geospatial_dist with config 5node-spec-planning: 29 tests, 0 failures
--- done: testdata/logic_test/merge_join_dist with config 5node-spec-planning: 12 tests, 0 failures
--- total progress: 139643 statements/queries
--- done: testdata/logic_test/contention_event with config 5node-spec-planning: 12 tests, 0 failures
E210326 02:44:36.612539 13260411 sql/stats/automatic_stats.go:302  [-] 1323  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/5node-spec-planning/inverted_join_json_array_dist (2.13s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2821309&tab=buildLog) on [master@4f0629742ff665f8f6a60be60a5ecc5e4c248085](https://github.com/cockroachdb/cockroach/commits/4f0629742ff665f8f6a60be60a5ecc5e4c248085):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic493037186
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic493037186
--- FAIL: TestLogic (1044.14s)
=== RUN   TestLogic/local/virtual_columns
=== PAUSE TestLogic/local/virtual_columns
=== CONT  TestLogic/local/virtual_columns
=== CONT  TestLogic/local/virtual_columns
    testcluster.go:1442: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        --- FAIL: TestLogic/local/virtual_columns (2.05s)
=== RUN   TestLogic/local
    --- FAIL: TestLogic/local (0.72s)
=== RUN   TestLogic/fakedist-spec-planning/values
=== PAUSE TestLogic/fakedist-spec-planning/values
=== CONT  TestLogic/fakedist-spec-planning/values
=== CONT  TestLogic/fakedist-spec-planning/values
    logic.go:3112: 
        testdata/logic_test/values:60: error while processing
    logic.go:3112: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n2 at 127.0.0.1:40969: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- progress: testdata/logic_test/alter_primary_key: 46 statements/queries
        --- FAIL: TestLogic/fakedist-spec-planning/values (1.37s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.52s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2833204&tab=buildLog) on [master@fe14895b103aae7f969ebf90763802629ad209c5](https://github.com/cockroachdb/cockroach/commits/fe14895b103aae7f969ebf90763802629ad209c5):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic204041013
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic204041013
--- FAIL: TestLogic (1159.87s)
=== RUN   TestLogic/5node/merge_join_dist
=== PAUSE TestLogic/5node/merge_join_dist
=== CONT  TestLogic/5node/merge_join_dist
--- done: testdata/logic_test/distsql_datetime with config 5node: 2 tests, 0 failures
E210331 06:10:42.002065 13234838 jobs/adopt.go:260  [n1] 1340  job 645859383651368961: adoption completed with error job-update: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
=== CONT  TestLogic/5node/merge_join_dist
    logic.go:2549: 
        
        testdata/logic_test/merge_join_dist:23: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:41973: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/merge_join_dist with config 5node: 6 tests, 1 failures
E210331 06:10:44.563527 13138693 sql/logictest/logic.go:3565  [-] 1347  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:41973: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
    logic.go:3108: 
        testdata/logic_test/merge_join_dist:23: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:41973: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
--- done: testdata/logic_test/explain_analyze_plans with config 5node: 20 tests, 0 failures
E210331 06:10:44.864557 13319459 jobs/adopt.go:260  [n1] 1348  job 645859393018658817: adoption completed with error job 645859393018658817: could not mark as reverting: load-job-query: collecting referenced descriptors: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer: log-job: node unavailable; try another peer
--- done: testdata/logic_test/merge_join_dist_vec with config 5node: 18 tests, 0 failures
--- done: testdata/logic_test/inverted_filter_json_array_dist with config 5node: 30 tests, 0 failures
--- done: testdata/logic_test/inverted_join_multi_column_dist with config 5node: 16 tests, 0 failures
--- progress: testdata/logic_test/distsql_stats: 36 statements/queries
E210331 06:10:45.879016 13346534 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 1349  making txn commit explicit failed for "unnamed" meta={id=94981804 pri=0.01052059 epo=0 ts=1617171045.866589131,0 min=1617171045.866589131,0 seq=2} lock=true stat=STAGING rts=1617171045.866589131,0 wto=false gul=1617171046.366589131,0 int=1 ifw=1: result is ambiguous (server shutdown)
--- progress: testdata/logic_test/distsql_agg: 13 statements/queries
--- done: testdata/logic_test/distsql_distinct_on with config 5node: 15 tests, 0 failures
--- progress: testdata/logic_test/ranges: 44 statements/queries
E210331 06:10:46.368722 13296163 sql/stats/automatic_stats.go:302  [-] 1350  failed to refresh stats: node unavailable; try another peer
--- progress: testdata/logic_test/distsql_merge_join: 21 statements/queries
        --- FAIL: TestLogic/5node/merge_join_dist (2.62s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2841253&tab=buildLog) on [master@0966b275e63816fa62c1c754cde87917b8703f3c](https://github.com/cockroachdb/cockroach/commits/0966b275e63816fa62c1c754cde87917b8703f3c):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic327959758
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic327959758
--- FAIL: TestLogic (1002.65s)
=== RUN   TestLogic/5node/distsql_distinct_on
=== PAUSE TestLogic/5node/distsql_distinct_on
=== CONT  TestLogic/5node/distsql_distinct_on
E210402 00:37:51.203104 12852922 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 1291  making txn commit explicit failed for "unnamed" meta={id=2e492b4e pri=0.01943498 epo=0 ts=1617323871.197022144,0 min=1617323871.197022144,0 seq=2} lock=true stat=STAGING rts=1617323871.197022144,0 wto=false gul=1617323871.697022144,0 int=1 ifw=1: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
E210402 00:37:51.203257 12814387 sql/stats/automatic_stats.go:302  [-] 1292  failed to refresh stats: node unavailable; try another peer
=== CONT  TestLogic/5node/distsql_distinct_on
    logic.go:2549: 
        
        testdata/logic_test/distsql_distinct_on:40: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:37697: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_distinct_on with config 5node: 5 tests, 1 failures
E210402 00:37:53.473743 12764848 sql/logictest/logic.go:3565  [-] 1298  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:37697: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
    logic.go:3108: 
        testdata/logic_test/distsql_distinct_on:40: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:37697: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
E210402 00:37:53.482002 12900054 sql/stats/automatic_stats.go:302  [-] 1299  failed to refresh stats: node unavailable; try another peer
--- done: testdata/logic_test/explain_analyze_plans with config 5node: 20 tests, 0 failures
E210402 00:37:53.641656 12930890 jobs/adopt.go:260  [n1] 1300  job 646360182333669377: adoption completed with error result is ambiguous (server shutdown)
--- total progress: 138557 statements/queries
--- done: testdata/logic_test/distsql_datetime with config 5node: 2 tests, 0 failures
E210402 00:37:54.128419 12943965 jobs/adopt.go:260  [n1] 1301  job 646360183931830273: adoption completed with error job 646360183931830273: could not mark as reverting: load-job-query: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer: log-job: node unavailable; try another peer
--- done: testdata/logic_test/merge_join_dist_vec with config 5node: 18 tests, 0 failures
E210402 00:37:54.452850 12955137 jobs/adopt.go:260  [n1] 1302  job 646360184983977985: adoption completed with error job 646360184983977985: could not mark as reverting: node unavailable; try another peer: log-job: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- done: testdata/logic_test/inverted_join_multi_column_dist with config 5node: 16 tests, 0 failures
--- done: testdata/logic_test/dist_vectorize with config 5node: 23 tests, 0 failures
E210402 00:37:54.761377 12963032 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 1303  making txn commit explicit failed for "unnamed" meta={id=5b85e50a pri=0.00187382 epo=0 ts=1617323874.751517273,0 min=1617323874.751517273,0 seq=2} lock=true stat=STAGING rts=1617323874.751517273,0 wto=false gul=1617323875.251517273,0 int=1 ifw=1: result is ambiguous (server shutdown)
E210402 00:37:54.837396 12964543 jobs/adopt.go:260  [n1] 1304  job 646360186273169409: adoption completed with error job-update: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer
--- progress: testdata/logic_test/distsql_stats: 43 statements/queries
--- progress: testdata/logic_test/distsql_numtables: 19 statements/queries
--- progress: testdata/logic_test/ranges: 60 statements/queries
        --- FAIL: TestLogic/5node/distsql_distinct_on (2.32s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2844672&tab=buildLog) on [master@0e3006351557d1d6dc4e2179a9cca1fd702b09f6](https://github.com/cockroachdb/cockroach/commits/0e3006351557d1d6dc4e2179a9cca1fd702b09f6):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic075015043
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic075015043
--- FAIL: TestLogic (1069.83s)
=== RUN   TestLogic/5node-metadata/merge_join_dist
=== PAUSE TestLogic/5node-metadata/merge_join_dist
=== CONT  TestLogic/5node-metadata/merge_join_dist
=== CONT  TestLogic/5node-metadata/merge_join_dist
    logic.go:2549: 
        
        testdata/logic_test/merge_join_dist:23: 
        expected success, but found
        (XXUUU) cannot up-replicate to s2; missing gossiped StoreDescriptor
--- done: testdata/logic_test/merge_join_dist with config 5node-metadata: 6 tests, 1 failures
E210403 01:31:48.006562 13515067 sql/logictest/logic.go:3565  [-] 1284  pq: cannot up-replicate to s2; missing gossiped StoreDescriptor
    logic.go:3108: 
        testdata/logic_test/merge_join_dist:23: error while processing
    logic.go:3108: pq: cannot up-replicate to s2; missing gossiped StoreDescriptor
E210403 01:31:48.010720 13544797 sql/stats/automatic_stats.go:302  [-] 1285  failed to refresh stats: node unavailable; try another peer
--- done: testdata/logic_test/distsql_subquery with config 5node-metadata: 7 tests, 0 failures
        --- FAIL: TestLogic/5node-metadata/merge_join_dist (1.71s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2846133&tab=buildLog) on [master@df07592e26735be7362ef2b815ee163ead00739e](https://github.com/cockroachdb/cockroach/commits/df07592e26735be7362ef2b815ee163ead00739e):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic754400025
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic754400025
--- FAIL: TestLogic (1000.24s)
=== RUN   TestLogic/5node-metadata/hash_join_dist
=== PAUSE TestLogic/5node-metadata/hash_join_dist
=== CONT  TestLogic/5node-metadata/hash_join_dist
=== CONT  TestLogic/5node-metadata/hash_join_dist
    logic.go:2549: 
        
        testdata/logic_test/hash_join_dist:21: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:42913: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/hash_join_dist with config 5node-metadata: 6 tests, 1 failures
E210403 06:33:10.065656 13134939 sql/logictest/logic.go:3565  [-] 1334  pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:42913: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3108: 
        testdata/logic_test/hash_join_dist:21: error while processing
    logic.go:3108: pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:42913: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/distsql_tighten_spans with config 5node-metadata: 40 tests, 0 failures
        --- FAIL: TestLogic/5node-metadata/hash_join_dist (2.22s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2857801&tab=buildLog) on [master@fe1327d0c2fe35a572586a8f8ee961a2bf19ffda](https://github.com/cockroachdb/cockroach/commits/fe1327d0c2fe35a572586a8f8ee961a2bf19ffda):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic292273562
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic292273562
--- FAIL: TestLogic (878.98s)
=== RUN   TestLogic/local/set
=== PAUSE TestLogic/local/set
=== CONT  TestLogic/local/set
E210407 18:34:37.818555 138782 jobs/adopt.go:260  [n1] 14  job 647987454466621441: adoption completed with error job 647987454466621441: could not mark as reverting: job-update: collecting referenced descriptors: failed to send RPC: sending to all replicas failed; last error: node unavailable; try another peer: log-job: node unavailable; try another peer
=== CONT  TestLogic/local/set
    logic.go:2549: 
        
        testdata/logic_test/set:297: SHOW statement_timeout
        expected success, but found
        (57014) query execution canceled due to statement timeout
        errors.go:218: in init()
E210407 18:34:39.605075 2706 sql/logictest/logic.go:3540  [-] 16  
E210407 18:34:39.605075 2706 sql/logictest/logic.go:3540  [-] 16 +pq: query execution canceled due to statement timeout
    logic.go:2282: 
         pq: query execution canceled due to statement timeout
--- done: testdata/logic_test/set with config local: 79 tests, 2 failures
E210407 18:34:39.605764 2706 sql/logictest/logic.go:3565  [-] 17  testdata/logic_test/set:302: too many errors encountered, skipping the rest of the input
    logic.go:3108: 
        testdata/logic_test/set:302: error while processing
    logic.go:3108: testdata/logic_test/set:302: too many errors encountered, skipping the rest of the input
--- progress: testdata/logic_test/cascade: 84 statements/queries
        --- FAIL: TestLogic/local/set (2.02s)
=== RUN   TestLogic/local
    --- FAIL: TestLogic/local (1.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/logictest).TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2858449&tab=buildLog) on [master@f2eb62496fae3c6a72ad071b14e18a268af30ba5](https://github.com/cockroachdb/cockroach/commits/f2eb62496fae3c6a72ad071b14e18a268af30ba5):

```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic375382853
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic375382853
--- FAIL: TestLogic (1072.86s)
=== RUN   TestLogic/fakedist-spec-planning/partial_index
=== PAUSE TestLogic/fakedist-spec-planning/partial_index
=== CONT  TestLogic/fakedist-spec-planning/partial_index
=== CONT  TestLogic/fakedist-spec-planning/partial_index
    testcluster.go:1468: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        --- FAIL: TestLogic/fakedist-spec-planning/partial_index (1.56s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.66s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61657 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

- #58269 sql/logictest: TestLogic failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestLogic.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[sql/logictest.TestLogic failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2864544&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2864544&tab=artifacts#/) on [master @ e1e10f99362e67771e00346fb0e006627eb67b09](https://github.com/cockroachdb/cockroach/commits/e1e10f99362e67771e00346fb0e006627eb67b09):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic891717717
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic891717717
--- FAIL: TestLogic (1059.19s)
=== RUN   TestLogic/fakedist-spec-planning/txn
=== PAUSE TestLogic/fakedist-spec-planning/txn
=== CONT  TestLogic/fakedist-spec-planning/txn
=== CONT  TestLogic/fakedist-spec-planning/txn
    logic.go:2549: 
        
        testdata/logic_test/txn:14: SELECT * FROM kv
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:44407: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210409 05:33:27.769439 11155149 sql/logictest/logic.go:3540  [-] 1200  
E210409 05:33:27.769439 11155149 sql/logictest/logic.go:3540  [-] 1200 +pq: failed to connect to n2 at 127.0.0.1:44407: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2282: 
         pq: failed to connect to n2 at 127.0.0.1:44407: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/txn with config fakedist-spec-planning: 4 tests, 2 failures
E210409 05:33:27.769918 11155149 sql/logictest/logic.go:3565  [-] 1201  testdata/logic_test/txn:19: too many errors encountered, skipping the rest of the input
    logic.go:3108: 
        testdata/logic_test/txn:19: error while processing
    logic.go:3108: testdata/logic_test/txn:19: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist-spec-planning/txn (1.41s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.55s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]</p>
</details>

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project)

<sub>[Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2877172&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2877172&tab=artifacts#/) on master @ [47e818cf8af59dcf886a987bc18307a3457f48a6](https://github.com/cockroachdb/cockroach/commits/47e818cf8af59dcf886a987bc18307a3457f48a6):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic620710975
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic620710975
--- FAIL: TestLogic (858.10s)
=== RUN   TestLogic/fakedist-disk/inflight_trace_spans
=== PAUSE TestLogic/fakedist-disk/inflight_trace_spans
=== CONT  TestLogic/fakedist-disk/inflight_trace_spans
=== CONT  TestLogic/fakedist-disk/inflight_trace_spans
    logic.go:2323: let $curr_trace_id = 4911345170376068134
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039  
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +  FROM current_trace_spans
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +expected:
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +    true
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +but found (query options: "") :
E210413 13:41:26.202572 8260816 sql/logictest/logic.go:3540  [-] 1039 +    false
    logic.go:2282: 
         
        testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
          FROM current_trace_spans
        expected:
            true
        but found (query options: "") :
            false
--- done: testdata/logic_test/inflight_trace_spans with config fakedist-disk: 9 tests, 1 failures
        --- FAIL: TestLogic/fakedist-disk/inflight_trace_spans (1.41s)
=== RUN   TestLogic/fakedist-disk
    --- FAIL: TestLogic/fakedist-disk (0.71s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2879950&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2879950&tab=artifacts#/) on master @ [1daf2df61aabf3e8d00d736fa4a843d05b596544](https://github.com/cockroachdb/cockroach/commits/1daf2df61aabf3e8d00d736fa4a843d05b596544):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic352869799
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3333: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic352869799
--- FAIL: TestLogic (781.29s)
=== RUN   TestLogic/fakedist-spec-planning/default
=== PAUSE TestLogic/fakedist-spec-planning/default
=== CONT  TestLogic/fakedist-spec-planning/default
=== CONT  TestLogic/fakedist-spec-planning/default
    testcluster.go:1468: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/fakedist-spec-planning/default (1.48s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.49s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2884287&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2884287&tab=artifacts#/) on master @ [ea88b676d0c49b77375e32a290bd94753f93af48](https://github.com/cockroachdb/cockroach/commits/ea88b676d0c49b77375e32a290bd94753f93af48):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic623977100
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3312: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic623977100
--- FAIL: TestLogic (834.88s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)
=== RUN   TestLogic/5node/experimental_distsql_planning_5node
=== PAUSE TestLogic/5node/experimental_distsql_planning_5node
=== CONT  TestLogic/5node/experimental_distsql_planning_5node
=== CONT  TestLogic/5node/experimental_distsql_planning_5node
    testcluster.go:1468: failed to create a status client because of initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/5node/experimental_distsql_planning_5node (2.26s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2884372&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2884372&tab=artifacts#/) on master @ [4d987b3f0788c98fa358715b336a1c55e254ee50](https://github.com/cockroachdb/cockroach/commits/4d987b3f0788c98fa358715b336a1c55e254ee50):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic879897068
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3312: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic879897068
--- FAIL: TestLogic (804.70s)
=== RUN   TestLogic/5node/merge_join_dist_vec
=== PAUSE TestLogic/5node/merge_join_dist_vec
=== CONT  TestLogic/5node/merge_join_dist_vec
=== CONT  TestLogic/5node/merge_join_dist_vec
    logic.go:2528: 
        
        testdata/logic_test/merge_join_dist_vec:23: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39429: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/merge_join_dist_vec with config 5node: 6 tests, 1 failures
E210414 23:31:38.614575 11381253 sql/logictest/logic.go:3544  [-] 1242  pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39429: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
    logic.go:3087: 
        testdata/logic_test/merge_join_dist_vec:23: error while processing
    logic.go:3087: pq: while carrying out changes [{ADD_VOTER n2,s2}]: failed to connect to n2 at 127.0.0.1:39429: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        --- FAIL: TestLogic/5node/merge_join_dist_vec (2.25s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2889210&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2889210&tab=artifacts#/) on master @ [660df848fcd8569392791f35b27ef5e849b35e2c](https://github.com/cockroachdb/cockroach/commits/660df848fcd8569392791f35b27ef5e849b35e2c):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic182394997
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3312: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic182394997
--- FAIL: TestLogic (761.93s)
=== RUN   TestLogic/fakedist-spec-planning/aggregate
=== PAUSE TestLogic/fakedist-spec-planning/aggregate
=== CONT  TestLogic/fakedist-spec-planning/aggregate
=== CONT  TestLogic/fakedist-spec-planning/aggregate
    testcluster.go:296: listen tcp 127.0.0.1:0: listen: address already in use
        --- FAIL: TestLogic/fakedist-spec-planning/aggregate (0.24s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.48s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2899562&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2899562&tab=artifacts#/) on master @ [bf6da8a40f6956bc88ee4a930ddc931f73dd5ce4](https://github.com/cockroachdb/cockroach/commits/bf6da8a40f6956bc88ee4a930ddc931f73dd5ce4):


```
=== CONT  TestLogic
    logic.go:3312: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic993746173
--- FAIL: TestLogic (1080.73s)
=== RUN   TestLogic/fakedist-spec-planning/target_names
=== PAUSE TestLogic/fakedist-spec-planning/target_names
=== CONT  TestLogic/fakedist-spec-planning/target_names
=== CONT  TestLogic/fakedist-spec-planning/target_names
    logic.go:2528: 
        
        testdata/logic_test/target_names:39: SELECT (pg_get_keywords()).word FROM t
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:44747: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210419 22:25:18.149024 11276971 sql/logictest/logic.go:3519  [-] 1169  
E210419 22:25:18.149024 11276971 sql/logictest/logic.go:3519  [-] 1169 +pq: failed to connect to n2 at 127.0.0.1:44747: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2261: 
         pq: failed to connect to n2 at 127.0.0.1:44747: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/target_names with config fakedist-spec-planning: 6 tests, 2 failures
E210419 22:25:18.149481 11276971 sql/logictest/logic.go:3544  [-] 1170  testdata/logic_test/target_names:44: too many errors encountered, skipping the rest of the input
    logic.go:3087: 
        testdata/logic_test/target_names:44: error while processing
    logic.go:3087: testdata/logic_test/target_names:44: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist-spec-planning/target_names (1.41s)
=== RUN   TestLogic/fakedist-spec-planning/cascade/OptDriven
=== CONT  TestLogic/fakedist-spec-planning/cascade/OptDriven
    logic.go:2528: 
        
        testdata/logic_test/cascade:15: SELECT * FROM child
        expected success, but found
        (XXUUU) failed to connect to n3 at 127.0.0.1:46129: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210419 22:25:36.054415 12385639 sql/logictest/logic.go:3519  [-] 1198  
E210419 22:25:36.054415 12385639 sql/logictest/logic.go:3519  [-] 1198 +pq: failed to connect to n3 at 127.0.0.1:46129: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2261: 
         pq: failed to connect to n3 at 127.0.0.1:46129: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210419 22:25:36.054706 12385639 sql/logictest/logic.go:3519  [-] 1199  
E210419 22:25:36.054706 12385639 sql/logictest/logic.go:3519  [-] 1199 +testdata/logic_test/cascade:23: too many errors encountered, skipping the rest of the input
    logic.go:1805: 
         testdata/logic_test/cascade:23: too many errors encountered, skipping the rest of the input
--- done: testdata/logic_test/cascade with config fakedist-spec-planning: 2 tests, 3 failures
--- done: testdata/logic_test/window with config fakedist-spec-planning: 397 tests, 0 failures
--- total progress: 131332 statements/queries
            --- FAIL: TestLogic/fakedist-spec-planning/cascade/OptDriven (0.68s)
=== RUN   TestLogic/fakedist-spec-planning/cascade
=== PAUSE TestLogic/fakedist-spec-planning/cascade
=== CONT  TestLogic/fakedist-spec-planning/cascade
        --- FAIL: TestLogic/fakedist-spec-planning/cascade (1.91s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.72s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2899759&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2899759&tab=artifacts#/) on master @ [1c103726de6994b26706244144d3d9f7dd6b1f58](https://github.com/cockroachdb/cockroach/commits/1c103726de6994b26706244144d3d9f7dd6b1f58):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic039425781
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3312: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic039425781
--- FAIL: TestLogic (987.15s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.58s)
=== RUN   TestLogic/fakedist-spec-planning/timestamp/timestamp_precision
=== CONT  TestLogic/fakedist-spec-planning/timestamp/timestamp_precision
    logic.go:2528: 
        
        testdata/logic_test/timestamp:100: SELECT * FROM timestamp_test ORDER BY id ASC
        expected success, but found
        (XXUUU) failed to connect to n3 at 127.0.0.1:40241: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210419 22:59:32.858674 11450222 sql/logictest/logic.go:3519  [-] 1125  
E210419 22:59:32.858674 11450222 sql/logictest/logic.go:3519  [-] 1125 +pq: failed to connect to n3 at 127.0.0.1:40241: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2261: 
         pq: failed to connect to n3 at 127.0.0.1:40241: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210419 22:59:32.859058 11450222 sql/logictest/logic.go:3519  [-] 1126  
E210419 22:59:32.859058 11450222 sql/logictest/logic.go:3519  [-] 1126 +testdata/logic_test/timestamp:111: too many errors encountered, skipping the rest of the input
    logic.go:1805: 
         testdata/logic_test/timestamp:111: too many errors encountered, skipping the rest of the input
--- done: testdata/logic_test/timestamp with config fakedist-spec-planning: 19 tests, 3 failures
            --- FAIL: TestLogic/fakedist-spec-planning/timestamp/timestamp_precision (0.50s)
=== RUN   TestLogic/fakedist-spec-planning/timestamp
=== PAUSE TestLogic/fakedist-spec-planning/timestamp
=== CONT  TestLogic/fakedist-spec-planning/timestamp
        --- FAIL: TestLogic/fakedist-spec-planning/timestamp (1.81s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2902335&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2902335&tab=artifacts#/) on master @ [bea4754bdf607ab050a72b7e2a3fe9dca0289d34](https://github.com/cockroachdb/cockroach/commits/bea4754bdf607ab050a72b7e2a3fe9dca0289d34):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic622773250
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3321: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic622773250
--- FAIL: TestLogic (1030.15s)
=== RUN   TestLogic/fakedist-spec-planning/computed
=== PAUSE TestLogic/fakedist-spec-planning/computed
=== CONT  TestLogic/fakedist-spec-planning/computed
--- done: testdata/logic_test/conditional with config fakedist-spec-planning: 9 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/computed
    logic.go:2537: 
        
        testdata/logic_test/computed:72: SELECT c FROM with_no_column_refs
        expected success, but found
        (XXUUU) failed to connect to n3 at 127.0.0.1:38231: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210420 12:38:46.870038 11122369 sql/logictest/logic.go:3528  [-] 1172  
E210420 12:38:46.870038 11122369 sql/logictest/logic.go:3528  [-] 1172 +pq: failed to connect to n3 at 127.0.0.1:38231: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2270: 
         pq: failed to connect to n3 at 127.0.0.1:38231: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/computed with config fakedist-spec-planning: 14 tests, 2 failures
E210420 12:38:46.870229 11122369 sql/logictest/logic.go:3553  [-] 1173  testdata/logic_test/computed:78: too many errors encountered, skipping the rest of the input
    logic.go:3096: 
        testdata/logic_test/computed:78: error while processing
    logic.go:3096: testdata/logic_test/computed:78: too many errors encountered, skipping the rest of the input
E210420 12:38:46.872919 11957392 sql/stats/automatic_stats.go:302  [-] 1174  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/fakedist-spec-planning/computed (1.60s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.66s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2913762&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2913762&tab=artifacts#/) on master @ [b4766e0bdb491de6020d5d2acaa3d20f9c339176](https://github.com/cockroachdb/cockroach/commits/b4766e0bdb491de6020d5d2acaa3d20f9c339176):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic692930075
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3350: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic692930075
--- FAIL: TestLogic (794.20s)
=== RUN   TestLogic/fakedist-vec-off/vectorize_unsupported
=== PAUSE TestLogic/fakedist-vec-off/vectorize_unsupported
=== CONT  TestLogic/fakedist-vec-off/vectorize_unsupported
--- done: testdata/logic_test/where with config fakedist-vec-off: 21 tests, 0 failures
--- done: testdata/logic_test/inverted_index with config fakedist-vec-off: 264 tests, 0 failures
=== CONT  TestLogic/fakedist-vec-off/vectorize_unsupported
    logic.go:2539: 
        
        testdata/logic_test/vectorize_unsupported:28: select (x/1) from t39417
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:35759: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210422 20:37:40.337643 5095761 sql/logictest/logic.go:3557  [-] 587  
E210422 20:37:40.337643 5095761 sql/logictest/logic.go:3557  [-] 587 +pq: failed to connect to n2 at 127.0.0.1:35759: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2272: 
         pq: failed to connect to n2 at 127.0.0.1:35759: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/vectorize_unsupported with config fakedist-vec-off: 6 tests, 2 failures
E210422 20:37:40.346935 5095761 sql/logictest/logic.go:3582  [-] 588  testdata/logic_test/vectorize_unsupported:33: too many errors encountered, skipping the rest of the input
    logic.go:3125: 
        testdata/logic_test/vectorize_unsupported:33: error while processing
    logic.go:3125: testdata/logic_test/vectorize_unsupported:33: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist-vec-off/vectorize_unsupported (1.83s)
=== RUN   TestLogic/fakedist-vec-off
    --- FAIL: TestLogic/fakedist-vec-off (0.60s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2919783&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2919783&tab=artifacts#/) on master @ [924935cc8ee14ff0c6d23aedd2a12673bf864808](https://github.com/cockroachdb/cockroach/commits/924935cc8ee14ff0c6d23aedd2a12673bf864808):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic599276416
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3350: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic599276416
--- FAIL: TestLogic (816.45s)
=== RUN   TestLogic/fakedist/serial/serial_rowid
=== CONT  TestLogic/fakedist/serial/serial_rowid
    logic.go:2539: 
        
        testdata/logic_test/serial:31: SELECT count(DISTINCT a), count(DISTINCT b), count(DISTINCT c) FROM serial
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:35655: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
        nodedialer.go:169: in dial()
E210424 06:04:35.253726 4748781 sql/logictest/logic.go:3557  [-] 533  
E210424 06:04:35.253726 4748781 sql/logictest/logic.go:3557  [-] 533 +pq: failed to connect to n2 at 127.0.0.1:35655: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
    logic.go:2272: 
         pq: failed to connect to n2 at 127.0.0.1:35655: initial connection heartbeat failed: rpc error: code = Unavailable desc = timed out waiting for server handshake
E210424 06:04:35.254062 4748781 sql/logictest/logic.go:3557  [-] 534  
E210424 06:04:35.254062 4748781 sql/logictest/logic.go:3557  [-] 534 +testdata/logic_test/serial:36: too many errors encountered, skipping the rest of the input
    logic.go:1816: 
         testdata/logic_test/serial:36: too many errors encountered, skipping the rest of the input
--- done: testdata/logic_test/serial with config fakedist: 5 tests, 3 failures
            --- FAIL: TestLogic/fakedist/serial/serial_rowid (0.31s)
=== RUN   TestLogic/fakedist/serial
=== PAUSE TestLogic/fakedist/serial
=== CONT  TestLogic/fakedist/serial
        --- FAIL: TestLogic/fakedist/serial (1.64s)
=== RUN   TestLogic/fakedist
    --- FAIL: TestLogic/fakedist (0.58s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2925863&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2925863&tab=artifacts#/) on master @ [05880958efb6fda9ab80ed16495d98922de3fbf6](https://github.com/cockroachdb/cockroach/commits/05880958efb6fda9ab80ed16495d98922de3fbf6):

Fatal error:

```
panic: expected 2 components to be closed, but found that only 0 were

```
Stack: 

```
goroutine 9453985 [running]:
github.com/cockroachdb/cockroach/pkg/sql/colexecerror.InternalError(...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/colexecerror/error.go:192
github.com/cockroachdb/cockroach/pkg/sql/colflow.(*vectorizedFlow).Cleanup(0xc146920aa0, 0x51d5e40, 0xc037fcf240)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/colflow/vectorized_flow.go:285 +0x505
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).Run.func8()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:373 +0x82
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execWithDistSQLEngine(0xc0c811c600, 0x51f4140, 0xc05fff0318, 0xc0c811cae0, 0x3, 0x5243ee0, 0xc08e468c00, 0xc109496401, 0xc0b0f19078, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:1055 +0x4cc
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).dispatchToExecutionEngine(0xc0c811c600, 0x51f4140, 0xc05fff0318, 0xc0c811cae0, 0x5243ee0, 0xc08e468c00, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:915 +0x75b
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmtInOpenState(0xc0c811c600, 0x51f4140, 0xc05fff0318, 0x5200700, 0xc02c975f90, 0xc0979d1bc0, 0x3c, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:659 +0xf86
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execStmt(0xc0c811c600, 0x51d5e40, 0xc0ce4d52c0, 0x5200700, 0xc02c975f90, 0xc0979d1bc0, 0x3c, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor_exec.go:122 +0xb14
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd.func1(0x5200700, 0xc02c975f90, 0xc0979d1bc0, 0x3c, 0x0, 0x0, 0xad5db37, 0xed8193acf, 0x0, 0xad5db37, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1527 +0x225
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).execCmd(0xc0c811c600, 0x51d5e40, 0xc0ce4d52c0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1529 +0x1457
github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor).run(0xc0c811c600, 0x51d5f00, 0xc08e468870, 0xc081bf7e00, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go:1450 +0x228
github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx.func1(0xc0c811c600, 0x51d5f00, 0xc08e468870, 0xc0b0f18960, 0xc08e4689c0, 0x5188200, 0xc0ce4d5140, 0xc02ad0a630, 0xc17f462d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:202 +0x65
created by github.com/cockroachdb/cockroach/pkg/sql.(*InternalExecutor).initConnEx
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/internal.go:201 +0x2f2
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestLogic/fakedist-disk/serial/serial_virtual_sequence
E210426 22:50:55.106794 9444974 jobs/adopt.go:260  [n1] 954  job 653417037656260609: adoption completed with error failed to ingest index entries during backfill: duplicate key value violates unique constraint "bar"
```
</p>
</details>
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929446&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929446&tab=artifacts#/) on master @ [29305bfb08116f38e10ca090347eea170899b919](https://github.com/cockroachdb/cockroach/commits/29305bfb08116f38e10ca090347eea170899b919):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic555034003
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic555034003
--- FAIL: TestLogic (817.89s)
=== RUN   TestLogic/fakedist/zigzag_join
=== PAUSE TestLogic/fakedist/zigzag_join
=== CONT  TestLogic/fakedist/zigzag_join
=== CONT  TestLogic/fakedist/zigzag_join
    logic.go:2540: 
        
        testdata/logic_test/zigzag_join:38: SELECT n,a,b FROM a WHERE a = 4 AND b = 1
        expected success, but found
        (XXUUU) failed to connect to n3 at 127.0.0.1:44899: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210427 21:45:54.071236 3454388 sql/logictest/logic.go:3558  [-] 479  
E210427 21:45:54.071236 3454388 sql/logictest/logic.go:3558  [-] 479 +pq: failed to connect to n3 at 127.0.0.1:44899: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2273: 
         pq: failed to connect to n3 at 127.0.0.1:44899: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/zigzag_join with config fakedist: 8 tests, 2 failures
E210427 21:45:54.072134 3454388 sql/logictest/logic.go:3583  [-] 480  testdata/logic_test/zigzag_join:43: too many errors encountered, skipping the rest of the input
    logic.go:3126: 
        testdata/logic_test/zigzag_join:43: error while processing
    logic.go:3126: testdata/logic_test/zigzag_join:43: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist/zigzag_join (1.81s)
=== RUN   TestLogic/fakedist
    --- FAIL: TestLogic/fakedist (0.75s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929528&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2929528&tab=artifacts#/) on master @ [f3286c1e1b8d2d44f0b5c92460c0d5acbfdd1610](https://github.com/cockroachdb/cockroach/commits/f3286c1e1b8d2d44f0b5c92460c0d5acbfdd1610):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic036720330
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic036720330
--- FAIL: TestLogic (828.11s)
=== RUN   TestLogic/5node-metadata/distsql_numtables
=== PAUSE TestLogic/5node-metadata/distsql_numtables
=== CONT  TestLogic/5node-metadata/distsql_numtables
=== CONT  TestLogic/5node-metadata/distsql_numtables
    logic.go:2540: 
        
        testdata/logic_test/distsql_numtables:21: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n4,s4}]: failed to connect to n4 at 127.0.0.1:35873: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_numtables with config 5node-metadata: 4 tests, 1 failures
E210427 22:20:54.582791 12005955 sql/logictest/logic.go:3583  [-] 1219  pq: while carrying out changes [{ADD_VOTER n4,s4}]: failed to connect to n4 at 127.0.0.1:35873: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3126: 
        testdata/logic_test/distsql_numtables:21: error while processing
    logic.go:3126: pq: while carrying out changes [{ADD_VOTER n4,s4}]: failed to connect to n4 at 127.0.0.1:35873: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/contention_event with config 5node-metadata: 12 tests, 0 failures
--- total progress: 140737 statements/queries
        --- FAIL: TestLogic/5node-metadata/distsql_numtables (2.22s)
=== RUN   TestLogic/5node-metadata
    --- FAIL: TestLogic/5node-metadata (0.00s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2930086&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2930086&tab=artifacts#/) on master @ [171b1bd0971171214c203ad615e40a651b71848a](https://github.com/cockroachdb/cockroach/commits/171b1bd0971171214c203ad615e40a651b71848a):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic231064638
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic231064638
--- FAIL: TestLogic (800.82s)
=== RUN   TestLogic/fakedist-disk/schema_change_in_txn
=== PAUSE TestLogic/fakedist-disk/schema_change_in_txn
=== CONT  TestLogic/fakedist-disk/schema_change_in_txn
=== CONT  TestLogic/fakedist-disk/schema_change_in_txn
    testcluster.go:135: condition failed to evaluate within 45s: unexpectedly found 1 active spans:
             0.000ms      0.000ms    === operation:colbatchscan
        goroutine 8364100 [running]:
        runtime/debug.Stack(0xc00356b860, 0x5152500, 0xc12816d700)
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x9f
        github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoon(0x53233a0, 0xc008bd1080, 0xc00356b860)
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/soon.go:36 +0x6d
        github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).stopServers(0xc087166000, 0x51de520, 0xc000072128)
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:135 +0x23d
        github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).Start.func2()
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:335 +0x45
        github.com/cockroachdb/cockroach/pkg/util/stop.CloserFn.Close(0xc0792f12e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:110 +0x25
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc06c0fb780, 0x51de520, 0xc000072128)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:479 +0x22f
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).shutdownCluster(0xc088faa800)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:1602 +0x1da
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).close(0xc088faa800)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:1237 +0x3d
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).runFile(0xc088faa800, 0xc000c8f200, 0x28, 0x450b498, 0xd, 0x3, 0x1, 0x44f2451, 0x2, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:3132 +0x189
        github.com/cockroachdb/cockroach/pkg/sql/logictest.RunLogicTestWithDefaultConfig.func1.1(0xc008bd1080)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:3324 +0x2de
        testing.tRunner(0xc008bd1080, 0xc01d91f440)
        	/usr/local/go/src/testing/testing.go:1123 +0xef
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1168 +0x2b3
--- progress: testdata/logic_test/tenant: 25 statements/queries
--- done: testdata/logic_test/tenant with config fakedist-disk: 27 tests, 0 failures
--- total progress: 122871 statements/queries
        --- FAIL: TestLogic/fakedist-disk/schema_change_in_txn (65.26s)
=== RUN   TestLogic/fakedist-disk
    --- FAIL: TestLogic/fakedist-disk (0.65s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2931741&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2931741&tab=artifacts#/) on master @ [112d0c111c2e9bd54cd13d40ad353cc4b286016e](https://github.com/cockroachdb/cockroach/commits/112d0c111c2e9bd54cd13d40ad353cc4b286016e):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic265796156
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic265796156
--- FAIL: TestLogic (1011.02s)
=== RUN   TestLogic/5node
    --- FAIL: TestLogic/5node (0.00s)
=== RUN   TestLogic/5node/distsql_distinct_on
=== PAUSE TestLogic/5node/distsql_distinct_on
=== CONT  TestLogic/5node/distsql_distinct_on
=== CONT  TestLogic/5node/distsql_distinct_on
    logic.go:2540: 
        
        testdata/logic_test/distsql_distinct_on:40: 
        expected success, but found
        (XXUUU) while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36903: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
--- done: testdata/logic_test/distsql_distinct_on with config 5node: 5 tests, 1 failures
E210428 11:16:37.957233 13281632 sql/logictest/logic.go:3583  [-] 1274  pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36903: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:3126: 
        testdata/logic_test/distsql_distinct_on:40: error while processing
    logic.go:3126: pq: while carrying out changes [{ADD_VOTER n3,s3}]: failed to connect to n3 at 127.0.0.1:36903: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        --- FAIL: TestLogic/5node/distsql_distinct_on (2.22s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932370&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932370&tab=artifacts#/) on master @ [7910db27b63ad68d2b43f618976ef03da5451ed2](https://github.com/cockroachdb/cockroach/commits/7910db27b63ad68d2b43f618976ef03da5451ed2):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic952021909
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic952021909
--- FAIL: TestLogic (931.44s)
=== RUN   TestLogic/fakedist-spec-planning/schema_change_in_txn
=== PAUSE TestLogic/fakedist-spec-planning/schema_change_in_txn
=== CONT  TestLogic/fakedist-spec-planning/schema_change_in_txn
--- progress: testdata/logic_test/join: 102 statements/queries
--- done: testdata/logic_test/select with config fakedist-spec-planning: 162 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/schema_change_in_txn
    testcluster.go:135: condition failed to evaluate within 45s: unexpectedly found 1 active spans:
             0.000ms      0.000ms    === operation:colbatchscan
        goroutine 10131596 [running]:
        runtime/debug.Stack(0xc07875b860, 0x5152860, 0xc3015f0fe0)
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x9f
        github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoon(0x5323840, 0xc03e975c80, 0xc07875b860)
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/soon.go:36 +0x6d
        github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).stopServers(0xc26c7c2480, 0x51de8c0, 0xc00019e018)
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:135 +0x23d
        github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).Start.func2()
        	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:335 +0x45
        github.com/cockroachdb/cockroach/pkg/util/stop.CloserFn.Close(0xc2fdab6c50)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:110 +0x25
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0365ad280, 0x51de8c0, 0xc00019e018)
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:479 +0x22f
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).shutdownCluster(0xc239594a00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:1602 +0x1da
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).close(0xc239594a00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:1237 +0x3d
        github.com/cockroachdb/cockroach/pkg/sql/logictest.(*logicTest).runFile(0xc239594a00, 0xc000ea5230, 0x28, 0x452aca3, 0x16, 0x3, 0x1, 0x44f25d1, 0x2, 0x0, ...)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:3132 +0x189
        github.com/cockroachdb/cockroach/pkg/sql/logictest.RunLogicTestWithDefaultConfig.func1.1(0xc03e975c80)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/logic.go:3324 +0x2de
        testing.tRunner(0xc03e975c80, 0xc1b4c981c0)
        	/usr/local/go/src/testing/testing.go:1123 +0xef
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1168 +0x2b3
        --- FAIL: TestLogic/fakedist-spec-planning/schema_change_in_txn (65.28s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.58s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932788&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932788&tab=artifacts#/) on master @ [852e22a2ad78b1a19bd4f504f7079274c0c95fe5](https://github.com/cockroachdb/cockroach/commits/852e22a2ad78b1a19bd4f504f7079274c0c95fe5):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic570632428
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic570632428
--- FAIL: TestLogic (1167.38s)
=== RUN   TestLogic/fakedist-spec-planning/poison_after_push
=== PAUSE TestLogic/fakedist-spec-planning/poison_after_push
=== CONT  TestLogic/fakedist-spec-planning/poison_after_push
--- progress: testdata/logic_test/fk: 252 statements/queries
--- done: testdata/logic_test/rename_constraint with config fakedist-spec-planning: 15 tests, 0 failures
=== CONT  TestLogic/fakedist-spec-planning/poison_after_push
    logic.go:2540: 
        
        testdata/logic_test/poison_after_push:35: SELECT * FROM t
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:33935: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210428 18:16:09.720477 11599440 sql/logictest/logic.go:3558  [-] 1206  
E210428 18:16:09.720477 11599440 sql/logictest/logic.go:3558  [-] 1206 +pq: failed to connect to n2 at 127.0.0.1:33935: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2273: 
         pq: failed to connect to n2 at 127.0.0.1:33935: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/poison_after_push with config fakedist-spec-planning: 7 tests, 2 failures
E210428 18:16:09.723213 11599440 sql/logictest/logic.go:3583  [-] 1207  testdata/logic_test/poison_after_push:40: too many errors encountered, skipping the rest of the input
    logic.go:3126: 
        testdata/logic_test/poison_after_push:40: error while processing
    logic.go:3126: testdata/logic_test/poison_after_push:40: too many errors encountered, skipping the rest of the input
        --- FAIL: TestLogic/fakedist-spec-planning/poison_after_push (1.92s)
=== RUN   TestLogic/fakedist-spec-planning
    --- FAIL: TestLogic/fakedist-spec-planning (0.67s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2936033&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2936033&tab=artifacts#/) on master @ [ea6bcca8816a2705e46481eeb7543b16bb7c258c](https://github.com/cockroachdb/cockroach/commits/ea6bcca8816a2705e46481eeb7543b16bb7c258c):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic134511204
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3351: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic134511204
--- FAIL: TestLogic (840.65s)
=== RUN   TestLogic/fakedist/poison_after_push
=== PAUSE TestLogic/fakedist/poison_after_push
=== CONT  TestLogic/fakedist/poison_after_push
=== CONT  TestLogic/fakedist/poison_after_push
    logic.go:2540: 
        
        testdata/logic_test/poison_after_push:35: SELECT * FROM t
        expected success, but found
        (XXUUU) failed to connect to n2 at 127.0.0.1:43475: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
        nodedialer.go:169: in dial()
E210429 07:43:04.114254 3494957 sql/logictest/logic.go:3558  [-] 511  
E210429 07:43:04.114254 3494957 sql/logictest/logic.go:3558  [-] 511 +pq: failed to connect to n2 at 127.0.0.1:43475: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
    logic.go:2273: 
         pq: failed to connect to n2 at 127.0.0.1:43475: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
--- done: testdata/logic_test/poison_after_push with config fakedist: 7 tests, 2 failures
E210429 07:43:04.115097 3494957 sql/logictest/logic.go:3583  [-] 512  testdata/logic_test/poison_after_push:40: too many errors encountered, skipping the rest of the input
    logic.go:3126: 
        testdata/logic_test/poison_after_push:40: error while processing
    logic.go:3126: testdata/logic_test/poison_after_push:40: too many errors encountered, skipping the rest of the input
E210429 07:43:04.118965 4588676 sql/stats/automatic_stats.go:302  [-] 513  failed to refresh stats: node unavailable; try another peer
        --- FAIL: TestLogic/fakedist/poison_after_push (1.42s)
=== RUN   TestLogic/fakedist/pg_extension
=== PAUSE TestLogic/fakedist/pg_extension
=== CONT  TestLogic/fakedist/pg_extension
E210429 07:43:02.638568 4568557 jobs/adopt.go:260  [n1] 500  job 654087889146118145: adoption completed with error could not remove enum value "hello" as it is being used by "use_greetings2" in row: k=1, v='hello'
--- done: testdata/logic_test/orms with config fakedist: 21 tests, 0 failures
=== CONT  TestLogic/fakedist/pg_extension
    logic.go:3130: 
        testdata/logic_test/pg_extension:55: error while processing
    logic.go:3130: test was successful but validation upon completion failed: failed to test for descriptor JSON round-trip: pq: failed to connect to n3 at 127.0.0.1:44877: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210429 07:43:04.106818 4588518 sql/stats/automatic_stats.go:302  [-] 509  failed to refresh stats: node unavailable; try another peer
E210429 07:43:04.113834 3494957 sql/logictest/logic.go:3571  [-] 510  
E210429 07:43:04.113834 3494957 sql/logictest/logic.go:3571  [-] 510 +testdata/logic_test/poison_after_push:35: SELECT * FROM t
E210429 07:43:04.113834 3494957 sql/logictest/logic.go:3571  [-] 510 +expected success, but found
E210429 07:43:04.113834 3494957 sql/logictest/logic.go:3571  [-] 510 +(XXUUU) failed to connect to n2 at 127.0.0.1:43475: initial connection heartbeat failed: rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: context deadline exceeded"
E210429 07:43:04.113834 3494957 sql/logictest/logic.go:3571  [-] 510 +nodedialer.go:169: in dial()
        --- FAIL: TestLogic/fakedist/pg_extension (1.57s)
=== RUN   TestLogic/fakedist
    --- FAIL: TestLogic/fakedist (0.76s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.1]
</p>
</details>
/cc @jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3116613&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3116613&tab=artifacts#/) on master @ [0604c999494816236a1ed8660405144f071e655b](https://github.com/cockroachdb/cockroach/commits/0604c999494816236a1ed8660405144f071e655b):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic738526644
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3355: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic738526644
--- FAIL: TestLogic (1221.08s)
=== RUN   TestLogic/local-vec-off/inflight_trace_spans
=== PAUSE TestLogic/local-vec-off/inflight_trace_spans
=== CONT  TestLogic/local-vec-off/inflight_trace_spans
=== CONT  TestLogic/local-vec-off/inflight_trace_spans
    logic.go:2317: let $curr_trace_id = 1139691570243500368
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170  
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +  FROM current_trace_spans
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +expected:
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +    true
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +but found (query options: "") :
E210624 19:44:03.657660 1390457 sql/logictest/logic.go:3563  [-] 170 +    false
    logic.go:2276: 
         
        testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
          FROM current_trace_spans
        expected:
            true
        but found (query options: "") :
            false
--- done: testdata/logic_test/inflight_trace_spans with config local-vec-off: 9 tests, 1 failures
        --- FAIL: TestLogic/local-vec-off/inflight_trace_spans (3.13s)
=== RUN   TestLogic/local-vec-off
    --- FAIL: TestLogic/local-vec-off (0.93s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot T-sql-schema branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-queries jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestLogic [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3212067&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3212067&tab=artifacts#/) on master @ [5989de86b019c154076e42eb0cac1a55ac5c0be3](https://github.com/cockroachdb/cockroach/commits/5989de86b019c154076e42eb0cac1a55ac5c0be3):


```
=== RUN   TestLogic
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic769877379
    test_log_scope.go:74: use -show-logs to present logs inline
=== CONT  TestLogic
    logic.go:3362: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic769877379
--- FAIL: TestLogic (1369.21s)
=== RUN   TestLogic/local/inflight_trace_spans
=== PAUSE TestLogic/local/inflight_trace_spans
=== CONT  TestLogic/local/inflight_trace_spans
--- done: testdata/logic_test/int_size with config local: 32 tests, 0 failures
--- progress: testdata/logic_test/vectorize: 155 statements/queries
--- progress: testdata/logic_test/role: 263 statements/queries
=== CONT  TestLogic/local/inflight_trace_spans
    logic.go:2324: let $curr_trace_id = 9111394917085194647
=== CONT  TestLogic/local/inflight_trace_spans
    logic.go:2283: 
         
        testdata/logic_test/inflight_trace_spans:44: SELECT count(*) = 0
          FROM current_trace_spans
        expected:
            true
        but found (query options: "") :
            false
--- done: testdata/logic_test/inflight_trace_spans with config local: 9 tests, 1 failures
        --- FAIL: TestLogic/local/inflight_trace_spans (3.20s)
=== RUN   TestLogic/local
    --- FAIL: TestLogic/local (1.29s)
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestLogic PKG=./pkg/sql/logictest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
<p>Parameters in this failure:

- GOFLAGS=-json
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63262 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-20.2]
- #61657 sql/logictest: TestLogic failed [C-test-failure O-robot branch-release-21.1]
- #58269 sql/logictest: TestLogic failed [C-test-failure O-robot T-sql-schema branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-queries jordanlewis
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/col/coldata/bytes.go
+++ b/pkg/col/coldata/bytes.go
@@ -492,6 +492,9 @@ func (b *Bytes) Truncate(length int) {
 		// This is a no-op.
 		return
 	}
+	// Ensure that calling Truncate with a length greater than maxSetLength does
+	// not invalidate the non-decreasing invariant.
+	b.UpdateOffsetsToBeNonDecreasing(length)
 	b.data = b.data[:b.offsets[length]]
 	b.maxSetLength = length
 }

--- a/pkg/sql/logictest/testdata/logic_test/window
+++ b/pkg/sql/logictest/testdata/logic_test/window
@@ -4279,3 +4279,25 @@ NULL  NULL  NULL
 
 statement ok
 RESET vectorize;
+
+# Regression test for incorrect bytes results when spilling to disk when there
+# are multiple partitions and one of the earlier partitions has trailing NULL
+# values.
+statement ok
+DROP TABLE IF EXISTS t;
+CREATE TABLE t (x INT, y STRING);
+INSERT INTO t VALUES (1, 'NotNull'), (1, NULL), (1, NULL),
+(2, 'NotNull'), (2, 'NotNull'), (2, 'NotNull'), (2, 'NotNull');
+
+# Loading a batch with trailing nulls onto disk and then unloading it to
+# continue processing should not corrupt the output bytes.
+query ITT rowsort
+SELECT x, y, first_value(y) OVER (PARTITION BY x ROWS BETWEEN CURRENT ROW AND CURRENT ROW) FROM t;
+----
+1  NotNull  NotNull
+1  NULL     NULL
+1  NULL     NULL
+2  NotNull  NotNull
+2  NotNull  NotNull
+2  NotNull  NotNull
+2  NotNull  NotNull

